keel-front50/src/main/kotlin/com/netflix/spinnaker/keel/front50/model/Application.kt

```diff
@@ -37,4 +39,35 @@ data class Application(
 
   @JsonAnyGetter
   fun details() = details
+
+  override fun toString(): String {
+    return "Application(name='$name', description='$description', email='$email', updateTs='$updateTs', createTs='$createTs', platformHealthOnly=$platformHealthOnly, platformHealthOnlyShowOverride=$platformHealthOnlyShowOverride, owner=$owner, details=$details)"
+  }
+
+  /*
+   * Need to identify additional ignored properties that are in
+   * the 'details' map
+   */
+  fun computedPropertiesToIgnore(): List<String> {
+    return listOf("user","lastModifiedBy","requiredGroupMembership")
+  }
+
+  /*
+   * Need to flatten all objects in details to top level objects to
+   * match the Application Spec
+   */
+  fun convertToMap(): MutableMap<String, Any?> {
```

Comment:
I wonder if this wouldn't make more sense being in the converter?

Reply:
I was torn. On the one hand I like to have the logic closely tied with the class, but on the other we do have a converter.. If it makes more sense to you to have it in the converter I will move it.

Reply:
Makes more sense to have it in the converter. I was torn on that as well, but if we're going to have a converter and that's where all other conversions are happening, seems to make sense to keep it all in one place.